parquet/src/arrow/arrow_reader/selection.rs

```diff
@@ -117,6 +117,39 @@ impl RowSelection {
         Self { selectors }
     }
 
+    /// Creates a [`RowSelection`] from a slice of uncombined `RowSelector`:
+    /// Like [skip(5),skip(5),read(10)].
+    /// After combine will return [skip(10),read(10)]
+    /// # Note
+    /// If directly use uncombined `RowSelection` with offset_index in parquet
+    /// will panic.
+    fn from_selectors_and_combine(selectors: &[RowSelector]) -> Self {
+        if selectors.len() < 2 {
+            return Self {
+                selectors: Vec::from(selectors),
+            };
+        }
+        let first = selectors.first().unwrap();
+        let mut sum_rows = first.row_count;
+        let mut selected = first.skip;
```

Comment:
```suggestion
        let mut skip = first.skip;
```

Maybe it would be easier to read this code if the name of the variable is the same as the name of the field that uses it 🤔

Reply:
Yes 😂